README.md

```diff
@@ -116,19 +116,20 @@ This classifier is a taken from our [tutorial on training a spiking MNIST classi
 ```python
 import torch, torch.nn as nn
 from norse.torch import LICell             # Leaky integrator
-from norse.torch import LIFFeedForwardCell # Leaky integrate-and-fire
+from norse.torch import LIFCell            # Leaky integrate-and-fire
 from norse.torch import SequentialState    # Stateful sequential layers
 
 model = SequentialState(
     nn.Conv2d(3, 20, 5, 1),      # Convolve from 3 -> 20 channels
-    LIFFeedForwardCell(),        # Spiking activation layer
+    LIFCell(),                   # Spiking activation layer
```

Comment:
Inconsistent with what you are writing in getting started below.

Reply:
Good catch. I haven't fully looked through the docs. I'll go through it now